README.md

```diff
@@ -1,2 +1,13 @@
 # C bindings for Skia
-These have been tailored to the needs of my specific projects and my not be suitable for anyone else.
\ No newline at end of file
+
+These have been tailored to the needs of my specific projects and my not be suitable for anyone else.
+
+# Skia version
+
+The Skia version with which these bindings have been tested is from _Wed Nov 10 21:38:56 2021 +0000_
+* https://skia.googlesource.com/skia/+/6fae0523629f9abf114d8b7413f71dc7295a13e0
+* https://chromium.googlesource.com/chromium/tools/depot_tools.git/+/4d3319e39c9b50747f47da35a351d428a38bcc82
+
+# Noticable changes in Skia code base reflected here
```

Comment:
Please make this and "Skia version" above second-level headers (i.e. add another #).

Also, could you add something like:

```
### November 10, 2021
```
above the list below, so that as more updates come out, folks can see what the delta was between the current and whatever date they last used.

Reply:
Done